cbsurge/components/builtenv/electricity/__init__.py

```diff
@@ -87,13 +87,15 @@ def get_url(self, dataset_url):
 
 
 
-class ElectricityVariable(Variable, ABC):
+class ElectricityVariable(Variable):
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
-        project = Project(path=os.getcwd())
+        #project = Project(path=os.getcwd())
 
     def download(self, **kwargs):
-        url = self.source
+        print(kwargs)
+        url = self.ge
+        print(self.source)
         # TODO: download data within polygons layer in geopackage
         self.download_geodata_by_admin(dataset_url=url, geopackage_path=self.geopackage_path)
```

Comment:
@Thuhaa to make this function works, please fix like below.

```
    def download(self, **kwargs):
        print(kwargs)
        url = self.source
        print(self.source)
        project = Project(path=os.getcwd())
        geopackage_path = project.geopackage_file_path
        # TODO: download data within polygons layer in geopackage
        self.download_geodata_by_admin(dataset_url=url, geopackage_path=geopackage_path)
```

but there is still an error inside